contracts/Bulker.sol

```diff
@@ -14,28 +14,28 @@ interface IClaimable {
 contract Bulker {
     /** General configuration constants **/
     address public immutable admin;
-    address payable public immutable weth;
+    address payable public immutable wrappedNativeAsset;
```

Comment:
Changing this value from `weth` to `wrappedNativeAsset` is purely a cosmetic change (the variable might actually refer to a different wrapped asset, like wrapped Matic or wrapped Avax).

It's a nice update, but causes issues when trying to read this value for deployments that use the existing Bulker (trying to read `bulker.wrappedNativeAsset()` from those contracts reverts, since their equivalent is `bulker.weth()`).

Some options to deal with this:

1. deploy new Bulkers for `mainnet` and `goerli`
2. revert these updates to Bulker; leave the references to `weth` as is
3. create a `context.getNativeAssetWrapper` function that returns the value at this storage slot, wether it's called `weth` or `wrappedNativeAsset`

re-deploying seems heavy-handed; 2 or 3 seem preferable

Reply:
also resolved in parking lot